drivers/backup/portworx/portworx.go

```diff
@@ -47,6 +47,8 @@ const (
 	defaultTimeout        = 5 * time.Minute
 )
 
+var UserContextRegisterCluster string = ""
```

Comment:
Same here. Think about a different variable name and need to assign `""` to the variable